### PR TITLE
PurchaseCmd 

### PR DESCRIPTION
**Pull request by brianmontero18**
PurchaseCmd is done, ready to review

### CODE REVIEW

**Comment by elin3t** [comment]
-1 is not possible to run the test with success!

```
perl -t purchase_cmd.t   
Can't locate object method "new" via package "UserService" (perhaps you forgot to load "UserService"?) at purchase_cmd.t line 10.
```

please fix it

**Comment by earandap** [comment]
OrderService is not ready yet.

**Comment by brianmontero18** [comment]
Fixed. The t file has been updated

File: lib/OrderService.pm [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    my ($user_id, $ord_num, $description, $amount_of_pkgs) = @_;
+    if($ord_num eq "95332"){
+        return Error->new("El pedido '95332' ya fue ingresado al sistema");
+    }elsif($user_id eq "jjlopez"){
+        return Output->new("Compra '$ord_num' registrada");
+    }
+    return Error->new("El usuario '$user_id' no fue encontrado");
+}
+
+1;
```
Comment (elin3t) [inline lib/OrderService.pm]: In this commit you must not push the file OrderService
@brianmontero18  please remove it

File: lib/OrderService.pm [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package OrderService;
```
Comment (pauriarte) [inline lib/OrderService.pm]: delete this OrderService package

File: lib/PurchaseCmd.pm [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+use v5.18;
+use parent qw(Command);
+
+use OrderService;
+
+sub execute {
+    my $self = shift;
+    my @parameters = $self->parameters();
+    my $order_service = OrderService->new();
+    return $order_service->buy($parameters[0], $parameters[1], $parameters[2], $parameters[3])
```
Comment (elin3t) [inline lib/PurchaseCmd.pm]: put ';' at the end of the line

File: lib/OrderService.pm (hunk 2 above)
Comment (manuellerena) [inline lib/OrderService.pm]: o.0 
I think this file doesn't should be here

File: lib/PurchaseCmd.pm [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#!/user/bin/perl
```
Comment (manuellerena) [inline lib/PurchaseCmd.pm:1]: Please see the approach to isolate the Command to the OrderService on the ReceptionCmd and receptioncmd.t

**Comment by brianmontero18** [comment]
I removed the ServiceOrder module and I replaced it by ServiceOrderFake whitin of t file

**Comment by manuellerena** [comment]
+1

**Comment by elin3t** [comment]
+1

**Pull request merged**